lib/rules/destructure-namespaces.js

```diff
@@ -20,4 +27,13 @@ var rule = function(context) {
   };
 };
 
+rule.schema = [
```

Comment:
Schema validates configuration passed to a rule. Unfortunately I was not able to find an example of testing config validation errors in the official ESLint rules that I looked at. The happy path is tested: you can configure the rule according to the tests. But I couldn't figure out a conventional path to expect errors and have a passing test.